cpp/core/compute/Backend.h

```diff
@@ -80,6 +82,16 @@ class Backend : public std::enable_shared_from_this<Backend> {
     return std::make_shared<ArrowColumnarToRowConverter>(rb, memory_pool);
   }
 
+  virtual std::shared_ptr<CommonSplitter> makeSplitter(
+      const std::string& partitioning_name,
+      int num_partitions,
+      SplitOptions options,
+      const std::string& batchType) {
+    GLUTEN_ASSIGN_OR_THROW(
+        auto splitter, Splitter::Make(partitioning_name, num_partitions, std::move(options)));
+    return nullptr;
```

Comment:
Is this intentional? Probably:
```suggestion
    return splitter;
```

Reply:
Forget here, you are right